<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HumanPlayerInterface</class>
 <widget class="QDialog" name="HumanPlayerInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>899</width>
    <height>424</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>899</width>
    <height>424</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>899</width>
    <height>424</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>200</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="horse_4_PositionLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>290</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="player_3_SecondHorseLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>260</y>
     <width>21</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>230</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="player_1_SecondHorseLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>200</y>
     <width>21</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>230</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Spēlētājs : 2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="horse_5_PositionLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>320</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>260</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>230</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1)</string>
   </property>
  </widget>
  <widget class="QSlider" name="horse_4_PositionSlider">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>293</y>
     <width>351</width>
     <height>19</height>
    </rect>
   </property>
   <property name="maximum">
    <number>50</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="horse_1_PositionSlider">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>203</y>
     <width>351</width>
     <height>19</height>
    </rect>
   </property>
   <property name="maximum">
    <number>50</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_26">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>260</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Spēlētājs : 3</string>
   </property>
  </widget>
  <widget class="QSlider" name="horse_5_PositionSlider">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>323</y>
     <width>351</width>
     <height>19</height>
    </rect>
   </property>
   <property name="maximum">
    <number>50</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>160</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Zirgu pozicijas:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="horse_3_PositionLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>260</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="player_3_FirstHorseLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>260</y>
     <width>21</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QSlider" name="horse_2_PositionSlider">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>233</y>
     <width>351</width>
     <height>19</height>
    </rect>
   </property>
   <property name="maximum">
    <number>50</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="player_1_FirstHorseLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>200</y>
     <width>21</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="horse_2_PositionLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>230</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QSlider" name="horse_3_PositionSlider">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>263</y>
     <width>351</width>
     <height>19</height>
    </rect>
   </property>
   <property name="maximum">
    <number>50</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>200</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>160</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Spēlētāju zirgi:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="player_2_SecondHorseLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>230</y>
     <width>21</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="horse_1_PositionLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>200</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>200</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Spēlētājs : 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Zirgi:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>260</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="player_2_FirstHorseLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>230</y>
     <width>21</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Kauliņš:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="diceLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Jūs esat spēlētājs :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="playerTurnLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>80</y>
     <width>21</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="horse_1_Button">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>195</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Zirgs : 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>20</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Spēlētāja saskarne</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>360</y>
     <width>431</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>Lūdzu, uzspiediet uz attiecīga zirga pogu, kuru Jūs gribat pārbīdīt.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>385</y>
     <width>431</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>Ja Jūs negribat turpināt spēli, lūdzu, pārtrauciet procesu.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="horse_2_Button">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>225</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Zirgs : 2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="horse_3_Button">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>255</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Zirgs : 3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="horse_4_Button">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>285</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Zirgs : 4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="horse_5_Button">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>315</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Zirgs : 5</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
